code/modules/atmospherics/components/binary_devices/binary_atmos_base.dm

```diff
@@ -132,26 +33,17 @@
 /obj/machinery/atmospherics/binary/wrench_floor_bolts(user)
 	. = ..()
 	if(anchored)
-		if(dir & (NORTH|SOUTH))
-			initialize_directions = NORTH|SOUTH
-		else if(dir & (EAST|WEST))
-			initialize_directions = EAST|WEST
-
+		set_dir(dir) // making sure
 		atmos_init()
+		for(var/obj/machinery/atmospherics/node in nodes)
```

Comment:
As briefly discussed, might be worth throwing `as anything` in all these nodes iterations.

Reply:
Done.